core/src/test/java/org/apache/calcite/test/JdbcTest.java

```diff
@@ -8217,6 +8217,75 @@ private void checkGetTimestamp(Connection con) throws SQLException {
         .returns("EXPR$0=[1, 1.1]\n");
   }
 
+  /** Test case for
+   * <a href="https://issues.apache.org/jira/browse/CALCITE-6032">[CALCITE-6032]
+   * NullPointerException in Reldecorrelator for a Multi level correlated subquery</a>. */
+  @Test void testMultiLevelDecorrelation() throws Exception {
+    String hsqldbMemUrl = "jdbc:hsqldb:mem:.";
+    Connection baseConnection = DriverManager.getConnection(hsqldbMemUrl);
+    Statement baseStmt = baseConnection.createStatement();
+    baseStmt.execute("create table invoice (c5633_485 integer, c5633_503\n"
```

Comment:
this code looks like it's machine generated.
This makes it hard to read by a human.